.orchestra/ci/ci-run.sh

```diff
@@ -129,9 +132,13 @@ fi
 
 REMOTE="$(git remote get-url origin | sed 's|^\([^:]*:\)\([^/]\)|\1/\2|')"
 GITLAB_ROOT="$(dirname "$(dirname "$REMOTE")")"
-echo "${BASE_USER_OPTIONS_YML//\%GITLAB_ROOT\%/$GITLAB_ROOT}" > "$USER_OPTIONS"
+USER_OPTIONS_YML="${BASE_USER_OPTIONS_YML//\%GITLAB_ROOT\%/$GITLAB_ROOT}"
+USER_OPTIONS_YML="${USER_OPTIONS_YML//\%PRIVATE_SOURCES_PLACEHOLDER\%/$PRIVATE_SOURCES_CLONE_URL}"
+USER_OPTIONS_YML="${USER_OPTIONS_YML//\%PRIVATE_BIN_ARCHIVES_PLACEHOLDER\%/$PRIVATE_BIN_ARCHIVES_CLONE_URL}"
+echo "${USER_OPTIONS_YML//\%GITLAB_ROOT\%/$GITLAB_ROOT}" > "$USER_OPTIONS"
```

Comment:
Comment what's going on here, it's very cryptic otherwise.

Reply:
Done